### ISSUE TITLE
Accessibility Issue 

### ISSUE THREAD

**Issue by playitsaiff** [microsoft/powerbi-client-react#62]
Between 480px-320px width some part of the action bar right icon buttons are getting cut. 

![image](https://user-images.githubusercontent.com/46973001/151360679-36d7dedc-e1d3-47be-b831-0d28ac2beef3.png)
![asdfa](https://user-images.githubusercontent.com/46973001/151360951-27255c81-1c74-450c-96f0-b0760e7a4f19.png)

**Comment by v-MadhavC** [comment]
Thank you for reporting this issue. We've identified the root cause of the issue and will add it to our backlog.

**Comment by ayeshurun** [comment]
Thank you for your feedback and for bringing this to our attention. We have reviewed your issue and would like to inform you that it has been added to our backlog for further tracking.

As a result, we are closing this GitHub issue. For any further updates or questions regarding this matter, we kindly ask you to continue the conversation through our official support channels:
- For bug reports and technical assistance: Visit the [Fabric Developer Community Forum](https://community.fabric.microsoft.com/t5/Developer/bd-p/Developer). Our team and community experts are ready to assist you.
- For additional support: Contact your account manager or reach out to the [Fabric Support Team](https://support.fabric.microsoft.com/en-us/support/).

This will ensure that your queries are addressed promptly and by the appropriate team.

We appreciate your understanding and cooperation.
